.gitpod.dockerfile

```diff
@@ -16,7 +16,7 @@ RUN sudo apt-get update \
     # Clear package info (to free space)
     && sudo rm -rf /var/lib/apt/lists/*
 
-ENV NODE_VERSION="12.14.1"
+ENV NODE_VERSION="16.16.0"
```

Comment:
16.17.0 would be the current latest LTS version for v16.

Reply:
Agreed.